src/app/defaultController.spec.js

```diff
@@ -0,0 +1,60 @@
+(function() {
+  'use strict';
+
+  describe('getDefaultController check', function(){
+    var state, dataKey, controller;
+
+    beforeEach( module('healthGuide'));
+    beforeEach( function(){
+      //controller = router.getDefaultController('default');
+    });
+    beforeEach( module( function($provide){
+      dataKey = {};
+      $provide.value('dataKey', dataKey);
+    }));
+
+
+    beforeEach( inject( function( ) {
+    }));
+
+
+
+    describe('values: ', function(){
```

Comment:
I still can not access getDefaultController(state) for testing.(8h+++ stend on this issue)